<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartTheme</class>
 <widget class="QMainWindow" name="ChartTheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ChartTheme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>881</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QComboBox" name="themeComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="animatedComboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="legendComboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Theme</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Animations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Legned</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="chartComboBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Charts</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
